<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NNCrystal Trainer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resources/icons.qrc">
    <normaloff>:/icons/app_icon</normaloff>:/icons/app_icon</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_overview">
       <attribute name="title">
        <string>&amp;Overview</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="OverviewWidget" name="widget" native="true"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_dataset">
       <attribute name="title">
        <string>&amp;Dataset</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="DatasetWidget" name="widget_2" native="true"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_config">
       <attribute name="title">
        <string>&amp;Configuration</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="ConfigWidget" name="widget_3" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_train">
       <attribute name="title">
        <string>&amp;Train</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_evaluation">
       <attribute name="title">
        <string>&amp;Evaluation</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>&amp;About</string>
    </property>
    <addaction name="action_About"/>
   </widget>
   <widget class="QMenu" name="menuDebug">
    <property name="title">
     <string>Debug</string>
    </property>
    <addaction name="actionShow_states"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuAbout"/>
   <addaction name="menuDebug"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Exit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionShow_states">
   <property name="text">
    <string>Display States</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OverviewWidget</class>
   <extends>QWidget</extends>
   <header>components/overview</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DatasetWidget</class>
   <extends>QWidget</extends>
   <header>components/dataset</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ConfigWidget</class>
   <extends>QWidget</extends>
   <header>components/config</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resources/icons.qrc"/>
 </resources>
 <connections/>
</ui>
